conf/test_motus.config

```diff
@@ -44,5 +44,8 @@ params {
     run_diamond                           = false
     run_krakenuniq                        = false
     run_motus                             = true
+    motus_save_mgc_read_counts            = true
+    motus_remove_ncbi_ids                 = true
+    motus_use_relative_abundance          = true
```

Comment:
I would argue these should be false, as we want it to work with taxpasta (once that's in)

Reply:
Right, I update them.

Reply:
You can merge once tests pass :)

Reply:
Something strange with linting so far, thanks for the comments :)